src/__tests__/withProps.test.js

```diff
@@ -39,4 +39,31 @@ describe('withProps', () => {
     )
     expect(wrapper.equals(<div />)).toBeTruthy()
   })
+
+  it("doesn't compact surrounded HOC that hoists statics", () => {
```

Comment:
This tests `isCompactable()` (test fails against `master` version of `utils/createCompactableHOC`)